sharpy/combat/micro_workers.py

```diff
@@ -9,6 +9,9 @@
 
 class MicroWorkers(MicroStep):
     def group_solve_combat(self, units: Units, current_command: Action) -> Action:
+        if self.move_type in {MoveType.DefensiveRetreat, MoveType.PanicRetreat, MoveType.Push}:
```

Comment:
need to import MoveType above I think

Reply:
Nice catch.

Reply:
I actually got rid of all the `import *` from my own code, because this is a mistake I make all the time, so having flake catch it incredibly helpful.